### DISCUSSION TITLE
MCS%40UCI/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#292] [Announcements]
# MCS%40UCI/

None

https://opencs.app/MCS%40UCI/

**Comment by shockylove** [comment]
MCS拒绝了， 建议转到Master of Embedded & Cyber-physical Systems (MECPS) 项目。虽然我的文书都是机器学习/LLM内容

**Reply by jimlue530** [reply]
基本上只要拒了都會轉勸MECPS，不管你的BG是不是符合該方向

**Comment by Ritazwonderland** [comment]
这个项目听说只录和本校教授有过connection的学生？不知道真假
